packages/design-system/src/styles/components/_Table.scss

```diff
@@ -105,102 +104,86 @@ $table-dense-padding: $spacer-half $spacer-2 !default;
     display: block;
   }
 
-  td,
-  th {
-    border: 0;
-    border-top: 1px solid;
-
-    /* autoprefixer grid: on */
-    display: grid;
-    // CSS grid doesn't support word-wrap. Use minmax as a workaround
-    // https://css-tricks.com/css-grid-in-ie-debunking-common-ie-grid-misconceptions/#fit-content-is-not-ie-friendly-but
-    grid-template-columns: minmax(0, $table-stacked-header-max-width) 1fr;
-    position: relative;
-
-    // The psuedo element content is generated to display the row header `data-title` for the Tablecell content
-    /* stylelint-disable-next-line selector-no-qualifying-type */
-    &[data-title]::before {
-      content: attr(data-title);
-      font-weight: bold;
-
-      /* autoprefixer grid: on */
-      grid-column: 1;
-      max-width: $table-stacked-header-max-width;
-      width: calc(100% - #{$spacer-2});
-    }
-
-    // TableCell content is wrapped in a span for IE11 grid support
-    > span {
-      // Fixes IE11 display bug https://css-tricks.com/css-grid-in-ie-debunking-common-ie-grid-misconceptions/
-      display: block;
+  thead {
```

Comment:
Why is this necessary? Isn't the styling for `thead tr` already handle visually hiding the `thead`?

Reply:
Thanks, this is not necessary, `thead tr` already handle visually hiding.